Source/WebCore/platform/ScrollbarsController.h

```diff
@@ -34,12 +34,14 @@ namespace WebCore {
 
 class Scrollbar;
 class ScrollableArea;
+class ScrollingCoordinator;
 
 class ScrollbarsController {
     WTF_MAKE_FAST_ALLOCATED;
     WTF_MAKE_NONCOPYABLE(ScrollbarsController);
 public:
-    static std::unique_ptr<ScrollbarsController> create(ScrollableArea&);
+    WEBCORE_EXPORT static std::unique_ptr<ScrollbarsController> create(ScrollableArea&);
+    WEBCORE_EXPORT static std::unique_ptr<ScrollbarsController> createProxy(ScrollableArea&, ScrollingCoordinator*);
```

Comment:
The concept of "proxy" objects is generally used only between processes, which is not something that WebCore should know about.